src/ScssAssetConverter.php

```diff
@@ -25,8 +25,22 @@ class ScssAssetConverter extends Component implements AssetConverterInterface
      * significantly degrade the performance.
      */
     public $forceConvert = false;
-
-    private $compiler;
+    
+    public $formatter = null;
```

Comment:
Please add a type annotation (`@var string`) and also some documentation to the user on how this property should be used.

Reply:
Added simple documentation